fecgov/fec-cms
# Feature/4791 hp pres map link

## Summary

- Resolves #4791 

Replacing the homepage callout for the presidential data so that it points to /data/ generally

### Required reviewers

1-2 UX, front-end? It's pretty straight-forward

## Impacted areas of the application

This PR only adds the new image and styles—nothing on the site should actually change until the changes outlined in #4821

## Screenshots

When _this_ PR goes live, there should be no visible changes.

Here's how it should look after the hp content edit is made:
![image](https://user-images.githubusercontent.com/26720877/128387935-02d5cf0d-d6ac-44e9-9527-8c1426efe194.png)


## Related PRs

None

## How to test

- pull the branch
- `npm run build`
- `./manage.py runserver`
- Check the [homepage](http://127.0.0.1:8000)
- Login to your localhost cms
- Edit the homepage
- Copying the current content and paste somewhere temporary might be an easy safety net
- Edit the homepage to be the code chunk at the bottom of #4821
- Preview or publish the homepage
- Check that the left callout is "Search all campaign finance data"
- Make sure the three callouts and their text look like Production
